docs/source/index.rst

```diff
@@ -137,6 +137,27 @@ installed. The :samp:`build_yaml` interface accepts the following arguments:
 
 * :samp:`dist_path`: The directory to output the package files.
 
```

Comment:
Is there intentionally a blank line here? The blank lines above were because there is a numbered list between

Reply:
Got it. I removed the blank line. ✅